charts/kminion/templates/serviceaccount.yaml

```diff
@@ -9,4 +9,10 @@ metadata:
   annotations:
     {{- toYaml . | nindent 4 }}
   {{- end }}
+{{- if .Values.serviceAccount.imagePullSecrets }}
```

Comment:
Can you add that with some comments to the values.yaml so that we have a default and a bit of documentation?

Reply:
And could you explain me what service account image pull secrets are good for - or how are they different than the `imagePullSecrets` at the root level? I haven't used service accounts with image pull secrets before I guess

Reply:
Ah - I missed https://github.com/cloudhut/kminion/blob/cc3d571a471a772b9e5b2a76830c9fb234f2775f/charts/kminion/templates/deployment.yaml#L29-L32

In general, imagePullSecrets at `ServiceAccount` level spares us from defining imagePullSecrets in all pod specs - you define the SA only and the imagePullSecrets attached to it are used to pull images.

That being said - I don't see too much value with my PR now - as the root/Deployment level imagePullSecrets is all we need in this case